docs/modules/nats.md

```diff
@@ -75,6 +75,15 @@ These arguments are passed to the NATS server when it starts, as part of the com
 [Passing arguments](../../modules/nats/examples_test.go) inside_block:withArguments
 <!--/codeinclude-->
 
+#### Custom configuration file
+
+- Not available until the next release of testcontainers-go <a href="https://github.com/testcontainers/testcontainers-go"><span class="tc-version">:material-tag: main</span></a>
+
+It's possible to pass a custom config file to NATS container using `nats.WithConfigFile(strings.NewReader(config))`. The `io.Reader` is passed as a `-config /etc/nats.conf` arguments to an entrypoint. 
```

Comment:
suggestion: clarify that a file is written to the container with the contents of the reader and not just passed directly, as that's a little confusing currently

Reply:
fixed in ff17ba124cb6cf9c72bfd04206630fd4afb6fb8a